includes/admin/class-wp-job-manager-promoted-jobs-admin.php

```diff
@@ -179,134 +177,38 @@ public function promoted_jobs_custom_columns( $column ) {
-				font-size: 14px;
-				list-style: none;
-				padding-left: 32px;
-				margin: 12px 0;
-			}
-			.promote-job-modal .promote-job-modal-price {
-				display: flex;
-				flex-direction: column;
-			}
-			.promote-job-modal .promote-job-modal-price .price-text {
-				font-size: 12px;
-				text-transform: uppercase;
-				color: #787c82;
-			}
-			.promote-job-modal .promote-job-modal-price span {
-				margin-top: 10px;
-				font-size: 36px;
-				font-weight: 700;
-			}
-			.promote-job-modal .promote-buttons-group .button {
-				padding: 10px 16px;
-				border-radius: 2px;
-				margin-right: 18px;
-				border: 0;
-			}
-			.promote-job-modal .promote-buttons-group .button-primary {
-				background: #2270b1;
-				color: #fff;
-			}
-			.promote-job-modal .promote-buttons-group .button-secondary {
-				background: #fff;
-				color: #2270B1;
-				border: 1px solid #2270B1;
-			}
-			@media screen and (max-width: 1000px) {
-				.promote-job-modal-column-right {
-					display: none;
-				}
-			}
-			@media screen and (max-width: 782px) {
-				.promote-job-modal {
-					padding: 0px 25px 25px;
-				}
-			}
-			.wpjm-dialog {
-				border: 0;
-				border-radius: 8px;
-			}
-			form.dialog {
-				display: flex;
-			}
-			form.dialog button.dialog-close {
-				margin: 10px 15px auto auto;
-				content: "";
-				background: none;
-				border: 0;
-				font-size: 0;
-			}
-			form.dialog button.dialog-close:after {
-				content: "\2715";
-				font-size: 20px;
-			}
-		</style>
-		<div class="promote-job-modal">
-			<div class="promote-job-modal-column-left">
-				<h2 class="promote-jobs-heading">
-					Promote your job on our partner network.
-				</h2>
+		$promote_template                 = get_option( 'promote-jobs-template', false );
+		$promote_jobs_template_next_check = get_option( '_promote-jobs-template_next_check' );
 
-				<div class="promote-job-modal-price">
-					<div class="price-text">Starting From</div>
-					<span>$83.00</span>
-				</div>
+		if ( ! $promote_jobs_template_next_check || $promote_jobs_template_next_check < time() ) {
+			$check_for_updated_template = true;
+		}
 
-				<ul class="promote-list">
-					<li class="promote-list-item">Your ad will get shared on our Partner Network</li>
-					<li class="promote-list-item">Featured on jobs.blog for 7 days</li>
-					<li class="promote-list-item">Featured on our weekly email blast</li>
-				</ul>
+		if ( $check_for_updated_template ) {
+			$response = wp_safe_remote_get( 'https://wpjobmanager.com/wp-json/promoted-jobs/v1/assets/promote-dialog/?lang=' . get_locale() );
```

Comment:
Maybe we could use `get_user_locale()` instead of `get_locale()`? So we get the locale that the user is effectively seeing?

Related slack thread: p1688599499810659-slack-C053397HN30

Reply:
done in 1d7504f0486c74fc3dff10dda1a1c0dcbb5a9e91